build/Helix/PrepareHelixPayload.ps1

```diff
@@ -39,11 +40,11 @@ Write-Host "netCoreAppVer = $netCoreAppVer"
 # Copy files from nuget packages
 Copy-Item "$nugetPackagesDir\Microsoft.Internal.WinUI.Helix.$winuiHelixVer\scripts\test\*" $payloadDir
 Copy-Item "$nugetPackagesDir\microsoft.windows.apps.test.$mitaVer\lib\netcoreapp2.1\*.dll" $payloadDir
-Copy-Item "$nugetPackagesDir\taef.redist.wlk.$taefVer\build\Binaries\$Platform\*" $payloadDir
-Copy-Item "$nugetPackagesDir\taef.redist.wlk.$taefVer\build\Binaries\$Platform\CoreClr\*" $payloadDir
-New-Item -ItemType Directory -Force -Path "$payloadDir\.NETCoreApp2.1\"
-Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.$netCoreAppVer\runtimes\win-$Platform\lib\netcoreapp2.1\*" "$payloadDir\.NETCoreApp2.1\"
-Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.$netCoreAppVer\runtimes\win-$Platform\native\*" "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\Microsoft.Taef.$taefVer\build\Binaries\$Platform\*" $payloadDir
+Copy-Item "$nugetPackagesDir\Microsoft.Taef.$taefVer\build\Binaries\$Platform\netstandard2.0\*" $payloadDir
+Copy-Item "$nugetPackagesDir\Microsoft.NETCore.App.Runtime.win-$Platform.$netCoreAppVer\runtimes\win-$Platform\lib\net5.0\*" $payloadDir
+Copy-Item "$nugetPackagesDir\Microsoft.NETCore.App.Runtime.win-$Platform.$netCoreAppVer\runtimes\win-$Platform\native\*" $payloadDir
```

Comment:
>$payloadDir [](http://example.com/codeflow?start=121&length=11)

Previously, I ran into issues doing this. When the .net runtime binaries are copied into the root directory instead of into ".NetCoreApp2.1" I hit an issue where the powershell scripts that we run would try to load these dlls which was causing failures.

So it is probably best if we continue to copy the runtime dlls into a subdir.